src/js/view/controls/controls.js

```diff
@@ -200,34 +205,23 @@ export default class Controls extends Events {
                     api.play(settingsInteraction);
                 }
             }
-
-            // Trigger userActive so that a dismissive click outside the player can hide the controlbar
-            const activeTimeout = (visible || isKeyEvent) ? 0 : ACTIVE_TIMEOUT;
-            this.userActive(activeTimeout);
-            lastState = state;
-
-            const settingsButton = this.controlbar.elements.settingsButton;
-
-            if (!visible && settingsButton) {
-                if (isKeyEvent) {
-                    settingsButton.element().focus();
-                } else if (evt) {
-                    focusFloatingElement();
```

Comment:
are we breaking anything by not respecting this previous `focusFloatingElement` call?

Reply:
See Todos in description